lib/chargify_api_ares/response_helper.rb

```diff
@@ -1,5 +1,19 @@
 module Chargify
   module ResponseHelper
+    def save
```

Comment:
@moklett Just tried real quick to move all of this over to base and it doesn't work out. It messes with the prefix options on subscription.